src/routes/entity.js

```diff
@@ -188,4 +173,43 @@ router.delete('/:entity_id', async (req, res) => {
 	return utils.response(res, code, message);
 });
 
+// links entity with list of contacts
```

Comment:
Perhaps there's a linking method you could write to use here and other controller